jwst/assign_wcs/util.py

```diff
@@ -807,6 +807,57 @@ def update_s_region_imaging(model):
         model.meta.wcsinfo.s_region = s_region
 
 
+def update_s_region_lrs(model, reference_files):
+    """
+    Update ``S_REGION`` using V2,V3 of the slit corners from reference file.
+
+    Parameters
+    ----------
+    model : DataModel
+        Input model
+    reference_files : list
+        List of reference files for assign_wcs.
+
+    Returns
+    -------
+    None
+
+        s_region for model is updated in place.
+
```

Comment:
No need to specify None return, the specification of s_region updated in place, if you feel it is important, can go in the summary

Reply:
@emolter I am trying very hard to get this PR merged ASAP. The unit test requires some thought because I have to set up a WCS with the correct information. I will open a separate JIRA ticket to add this and track it.

Reply:
That's fine by me